velox/expression/tests/ExpressionFuzzer.cpp

```diff
@@ -405,6 +408,17 @@ class ExpressionFuzzer {
     }
   }
 
+  void appendConjunctSignatures() {
+    CallableSignature conjunctAndSignature;
+    conjunctAndSignature.name = "and";
+    conjunctAndSignature.returnType = BOOLEAN();
+    conjunctAndSignature.args = {BOOLEAN(), BOOLEAN()};
```

Comment:
@kagamiori @tanjialiang AND and OR have variadic number of arguments. Would it be possible to express that?

Reply:
I can work on that